templates/taxi/manufacturer_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+    <h1>Delete Manufacturer</h1>
+    <p>Are you sure thet you wont delete manufacturer "{{ manufacturer }}"?</p>
```

Comment:
There are typographical errors in this line. It should be: 'Are you sure that you want to delete manufacturer "{{ manufacturer }}"?'